t/lib/t/MusicBrainz/Server/Edit/Artist/Create.pm

```diff
@@ -68,7 +69,7 @@ SQL
     is($edit->display_data->{gender}->{name}, 'Male', "Gender is correct in display data.");
     is($edit->display_data->{comment}, 'Canadian electronica duo', "Comment is correct in display data.");
     is($edit->display_data->{begin_date}->format, "1981-05", "Begin date is correct in display data." );
-    is($edit->display_data->{ended}, 1, "Has 'ended' set in display data.");
+    is($edit->display_data->{ended}, boolean_to_json(1), "Has 'ended' set in display data.");
```

Comment:
I wasn't sure what to compare with here, so I decided we might as well just compare by using boolean_to_json here as well. But we might want to put exactly the expected output of boolean_to_json here in case boolean_to_json itself breaks... which would be what exactly? :)

Reply:
This seems fine but to answer your question, `JSON::true` or `\1` both work. :)